src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -218,18 +218,22 @@ public bool TryWriteProperties(PropertiesWriter writer, IList<ProjectInfo> proje
         {
             projectData.Status = ProjectInfoValidity.InvalidGuid;
         }
+        else if (!File.Exists(projectData.Project.FullPath)) // If a project was created and destroyed during the build, it is no longer valid
+        {
+            projectData.Status = ProjectInfoValidity.NoProjectFound;
+        }
```

Comment:
The new guy in the block.
Let me know if you want the comment to be more descriptive.